all_repos/sed.py

```diff
@@ -83,7 +84,14 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         dash_r = ('-r',)
     else:
         dash_r = ()
-    sed_cmd = ('sed', '-i', *dash_r, args.expression)
+
+    inplace_ext: Tuple[str, ...]
+    if sys.platform == 'darwin':  # pragma: no cover
```

Comment:
please don't change this, the README addresses this

also it's unrelated to your change